jenkins/settings.xml

```diff
@@ -50,6 +50,17 @@
           <name>sw-spark-maven</name>
           <url>${env.URM_URL}</url>
         </repository>
+        <!-- Add Sonatype staging repo for buiding rapids against pre-release jars  -->
+        <repository>
+           <id>sonatype-staging</id>
+           <url>https://oss.sonatype.org/content/repositories/staging</url>
+           <releases>
+             <enabled>true</enabled>
+           </releases>
+           <snapshots>
+             <enabled>false</enabled>
```

Comment:
I'm confused.  If it's a pre-release jar, isn't that the definition of a snapshot?

Reply:
This is not a snapshot jar, actually it's a pre-release jar, which is uploaded to sonatype staging repo.
Before we release cuDF before our plugin, so the cuDF jar should exist on public maven repo.
For 0.3, the cuDF jar will be built just one day before our plugin. So, I hope to release both after QA sign off with rapids plugin v0.3.

Reply:
Doesn't that mean we technically are not testing the build against the official cudf release in Maven central then?